Project.toml

```diff
@@ -18,9 +18,9 @@ CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 oneAPI = "8f75cd03-7ff8-4ecb-9b8f-daf728133b1b"
 
 [extensions]
-JACCAMDGPU = ["AMDGPU"]
-JACCCUDA = ["CUDA"]
-JACCONEAPI = ["oneAPI"]
+AMDGPUExt = ["AMDGPU"]
```

Comment:
Should we add JACC since @miniskar is planning to call these directly making them no longer "private"

Reply:
First, I don't think using the extensions directly is necessary. I think we resolved that by adding the LaunchSpec and synchronize bits. And we can add more functionality if necessary.

Second, if it IS necessary to access the extension. You would do that with `Base.get_extension(JACC, :AMDGPUExt)`, so the relationship would still be clear.

Reply:
Makes sense, thanks for the explanation!